Source/WebCore/page/Page.cpp

```diff
@@ -2013,6 +2022,9 @@ void Page::renderingUpdateCompleted()
         scheduleRenderingUpdateInternal();
         m_unfulfilledRequestedSteps = { };
     }
+
+    if (!isUtilityPage() && m_displayNominalFramesPerSecond)
```

Comment:
`m_displayNominalFramesPerSecond` can be 120fps on a ProMotion display but we'll only run rendering updates at `preferredRenderingUpdateFramesPerSecond()`, which is probably what you want to use here?

Reply:
Good point!

(I also noticed `preferredRenderingUpdateInterval` — would that be more appropriate here, or should I stick to `preferredRenderingUpdateFramesPerSecond`?)

Reply:
Those are equivalent (one is FPS, one is frame duration)